docs/source/resources/define-metadata.rst

```diff
@@ -1554,9 +1554,16 @@ information displays in the Anaconda.org channel.
 
   about:
     home: https://github.com/ilanschnell/bsdiff4
-    license: BSD
+    license: BSD 3-Clause
     license_file: LICENSE
-    summary: binary diff and patch using the BSDIFF4-format
+    license_family: BSD
+    summary: binary diff and patch using the BSDIFF4 format
+    description: |
+      This module provides an interface to the BSDIFF4 format, command line interfaces
+      (bsdiff4, bspatch4) and tests.
+    dev_url: https://github.com/ilanschnell/bsdiff4
+    doc_url: https://bsdiff4.readthedocs.io
+    doc_source_url: https://github.com/ilanschnell/bsdiff4/blob/main/README.rst
```

Comment:
Should we add `license_url` too?

```suggestion
    doc_source_url: https://github.com/ilanschnell/bsdiff4/blob/main/README.rst
    license_url: https://github.com/ilanschnell/bsdiff4/blob/master/LICENSE
```

Reply:
Yes. https://github.com/conda/conda-build/pull/5250